src/components/table/tableWrappers/OrdinancesTable.tsx

```diff
@@ -22,19 +22,24 @@ import { OrdinanceController } from "@/controllers/OrdinanceController";
 import ConfirmDialog from "@/components/common/ConfirmDialog";
 import { useState } from "react";
 import { PlayIcon } from "@heroicons/react/24/outline";
+import { SchoolTypeValues } from "@/entities/School";
 
 export default function OrdinancesTable({
   cityCode,
   founderId,
   initialData,
   count,
   urlFrom,
+  schoolType,
+  rootPath,
 }: {
   cityCode: number;
   founderId: number;
   initialData: any[];
   count?: number;
   urlFrom?: string[];
+  schoolType: string;

```

Comment:
In general, I don't like passing the `schoolType` as a string here. Also, I think the `rootPath` is a little redundant - you can derive it from the school type. If you add a `getRootPathBySchoolType(schoolType: SchoolType)` util function, you can simply replace the `${rootPath}` below by `getRootPathBySchoolType(schoolType)`. The code would become a little simpler and cleaner.